### DISCUSSION TITLE
Characters with accents or characters such as Ñ cannot be displayed

### DISCUSSION THREAD

**Discussion by rogerrey** [ruffle-rs/ruffle#2783] [Q&A]
Hi. 
I have verified that in some applications the characters with accents or characters such as Ñ cannot be displayed. It also happens in the text input fields. Is there any way to fix it?
Example: https://genmagic.net/visualurl.html?id=prueba_acentos.swf

Thanks a lot beforehand!

**Comment by ousia** [comment]
@rogerrey,

issue #1862 explains what may be the missing part here.

The SWF file relies on a “device font” to display the non-English characters.

In Ruffle, this font deployed doesn’t include accented characters (among many other ones).

From what #1862 explains, if the input text used an embedded font (which contains the glyphs), the accented characters would be displayed.

**Comment by ousia** [comment]
Since SWF files cannot be attached here, I simply added the `.zip` extension: [editcolors.swf.zip](https://github.com/ruffle-rs/ruffle/files/5862209/editcolors.swf.zip).

This is a standard SWF file without compression. Download it and drag and drop it at https://ruffle.rs/demo/.

Click on the center of the blank space and type in red only the embedded glyphs which are displayed in black.

As a suggestion (after years dealing with PDF documents), I would always embed fonts in my SWF files.

Relying on system fonts is complex at least.

**Comment by ousia** [comment]
#6224 fixed this. Update your selfhosted build and everything will work fine.
